src/views/blocks/ImageBlock.php

```diff
@@ -8,19 +8,38 @@
  * @param $this->varValue('align');
  * @param $this->varValue('fluid');
  * @param $this->varValue('image');
+ * @param $this->cfgValue('lazyload');
  *
  * @var $this \luya\cms\base\PhpBlockView
  */
+use luya\lazyload\LazyLoad;
 
 $align = $this->varValue('align', 'left');
 ?>
 
-<?php if ($this->extraValue('image', false)): ?>
+<?php if ($image = $this->extraValue('image', false)): ?>
     <div class="image">
         <figure class="figure d-block <?php if ($align === 'left'): ?>text-left<?php elseif ($align === 'right'): ?>text-right<?php else: ?>text-center<?php endif; ?>">
-            <img src="<?= $this->extraValue('image')->source ?>" class="figure-img img-fluid" <?php if ($this->extraValue('image')->caption): ?>alt="<?= $this->extraValue('image')->caption ?>"<?php endif; ?>>
-            <?php if ($this->varValue('showCaption', false) && $this->extraValue('image')->caption): ?>
-                <figcaption class="figure-caption"><?= $this->extraValue('image')->caption ?></figcaption>
+            <?php
+            $caption = !empty($image->caption) ? $image->caption : '';
+            if ($this->cfgValue('lazyload', false)):
```

Comment:
when using the php template syntax we usually close the php tag otherwise don't use template syntax.

Either `if (...): ?>` or `if (...) { ...... }`